samples/CommunityToolkit.Maui.Sample/App.xaml

```diff
@@ -31,26 +31,26 @@
                 <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource AppBackgroundLightColor}, Dark={StaticResource AppBackgroundDarkColor}}" />
             </Style>
 
-            <Style TargetType="Button">
+            <Style TargetType="Button" ApplyToDerivedTypes="true">
```

Comment:
What do you think if we move these styles to Styles.xaml, as in the .NET MAUI template, or even replace them with .NET MAUI styles?

Reply:
Sure!